<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainW</class>
 <widget class="QWidget" name="MainW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>378</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>378</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>378</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Yu Gothic UI Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="TitleLabel">
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="TestModeLabel">
         <property name="maximumSize">
          <size>
           <width>326</width>
           <height>26</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Test Mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="TestModeLayout">
       <item>
        <widget class="QRadioButton" name="StaticRdoButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Static Test</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ModeButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="DynamicRdoButton">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Dynamic Test</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ModeButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="DebugRdoButton">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Debug Test</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ModeButtonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="DebugModeLayout">
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="DebugLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Debug Mode</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QComboBox" name="DebugModeComboBox">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <item>
          <property name="text">
           <string>Static</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dynamic</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="NetworkCardLayout">
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="IfaceLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Network Card</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QComboBox" name="NetworkCardComboBox">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="LocalMacLayout">
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="IfaceLabel_2">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Local Mac Address</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QComboBox" name="LocalMacComboBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="DataPathLayout">
       <item>
        <widget class="QLabel" name="DataPathLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Data Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DataPathLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>19</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Please input the path of test data</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_1">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ResultFileLayout">
       <item>
        <widget class="QLabel" name="DataPathLabel_2">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Result File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ResFileLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>19</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Please input the path of result output</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="RunButton">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ModeButtonGroup"/>
 </buttongroups>
</ui>
